src/lib/functions/scheduled.js

```diff
@@ -0,0 +1,78 @@
+const ansi2html = require('ansi2html')
+
+const { CLOCKWORK_USERAGENT } = require('../../utils')
+
+const { formatLambdaError } = require('./utils')
+
+const handleScheduledFunction = ({ error, request, response, result }) => {
```

Comment:
could you please write a small unit test for this?

Reply:
done! 
6015fd6